dist/amd/defaultTranslations/relative.time.js

```diff
@@ -55,6 +55,35 @@ define(['exports'], function (exports) {
         'year_in_plural': 'in __count__ years'
       }
     },
+    es: {
```

Comment:
please revert the changes in generated files (under dist folder) - they are updated with release process.